Description: Added patch for two POD typos in Plugin.pm. Fixed upstream in
 r1673
Origin: other, http://bugs.debian.org/488700
Bug-Debian: http://bugs.debian.org/488700
Index: munin-1.2.6/node/Plugin.pm.in
===================================================================
--- munin-1.2.6.orig/node/Plugin.pm.in	2008-07-07 13:30:29.000000000 +0200
+++ munin-1.2.6/node/Plugin.pm.in	2008-07-07 13:30:51.000000000 +0200
@@ -303,7 +303,7 @@
 }
 
 
-=head ($file_handle,$rotated) = tail_open($file_name,$position)
+=head3 ($file_handle,$rotated) = tail_open($file_name,$position)
 
 Open the given file and seek to the given position.  If this position
 is beyond the end of the file the function assumes that the file has
@@ -352,7 +352,7 @@
     return ($fh,$filereset);
 }
 
-=head $position = tail_close($file_handle)
+=head3 $position = tail_close($file_handle)
 
 Close the the file and return the current position in the file.  This
 position should be put in a state vector and stored in a state file
